docs/ota-client-guide/modules/ROOT/pages/debugging-tips.adoc

```diff
@@ -12,11 +12,11 @@ For VS code, there is a `launch.json` provided in the `.vscode` directory.  This
 
 == Creating a temporary OSTree environment
 
-Try the scripts available in the link:../tests/ostree-scripts[tests/ostree-scripts] directory.
+Try the scripts available in the link:../../../../../tests/ostree-scripts[tests/ostree-scripts] directory.
```

Comment:
Relative links aren't appropriate here. It should be a fully qualified link to https://github.com/advancedtelematic/aktualizr/tree/{branch or tag}/tests